core/schema.mcf

```diff
@@ -11040,3 +11040,20 @@ name: "dateRevised"
 rangeIncludes: schema:Date,schema:DateTime
 domainIncludes: schema:Thing
 description: "The date on which the CreativeWork was last edited or revised or the item was last revised on a DataFeed."
+
+Node: dcid:AmenityFeatureEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "AmenityFeatureEnum"
+
+Node: dcid:Library
+typeOf: dcs:AmenityFeatureEnum
+name: "Library"
+
```

Comment:
Should we add these to dcschema.mcf?  Also please consider creating schema/stat-var MCF changes in critique/google3.

Reply:
Will do.